2020_12_03_array_list/src/OurListTest.java

```diff
@@ -423,16 +424,31 @@ public void testBackwardIterator_severalElements() {
             iterator.next();
         });
     }
+
     @Test
-    public void sort(){
+    public void sort() {
         integerOurList.addLast(6);
         integerOurList.addLast(28);
         integerOurList.addLast(9);
         integerOurList.addLast(15);
 
 
         integerOurList.sort(integerComparator);
-        assertEquals(28,integerOurList.get(3));
+        assertEquals(28, integerOurList.get(3));
+    }
+
+    @Test
+    public void sort_comparable() {
```

Comment:
design a test which checks the order of all elements. Meaning, check every element one by one so that they all are at their own places.